<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelView2Class</class>
 <widget class="QMainWindow" name="ModelView2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelView2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Add act:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="actlineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addpushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Add description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="descriptionlineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="acceptpushButton">
         <property name="text">
          <string>Accept idea</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ModelView2.qrc"/>
 </resources>
 <connections/>
</ui>
